tests/unit/identities/test_identities_models.py

```diff
@@ -44,10 +44,11 @@ def test_update_traits_remove_traits_with_none_value(identity_in_segment):
     trait_to_remove = TraitModel(trait_key=trait_key, trait_value=None)
 
     # When
-    updated_traits = identity_in_segment.update_traits([trait_to_remove])
+    updated_traits, was_updated = identity_in_segment.update_traits([trait_to_remove])
```

Comment:
Being really picky now but it's probably best to use `traits_updated` here. I'd say that, grammatically, this should be `were_updated` but then it's still not _immediately_ clear what were updated so let's be explicit?

Reply:
Yep, good point